### PR TITLE
Draft of user/customer guide for calling on consulting services.

### PR DESCRIPTION
**Pull request by warmanaMOJ**
As per subject line.

### CODE REVIEW

File: guides/user-guide.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## What happens next
+
+If your request is not appropriate for the consulting team, you'll be told immediately after the initial assessment concludes.
+
+If your request is appropriate for the consulting team, you'll be contacted directly by the assigned consultant, who will engage directly with you to start providing the help you need.
+
+## If things go wrong...
+
+If you disagree with a decision that your request is not appropriate for the consulting team, or there is some other problem, please contact the team again through the [cyberconsultancy@digital.justice.gov.uk](mailto:cyberconsultancy@digital.justice.gov.uk) email address. If you would prefer a different escalation route, please contact <img src="https://s3-eu-west-2.amazonaws.com/intranet-prod-storage-1dvcquh7kophi/uploads/2019/02/7d4181b72d6583785cd81bc9726d50c9.gif" alt="Alternative contact"> directly.
```
Comment (joelgsamuel) [inline guides/user-guide.md]: I feel like this should not be Jon's own email, more of a 'role' email address (aliases to the incumbent) `ciso@digital.justice.gov.uk` etc

**Comment by warmanaMOJ** [comment]
@joelgsamuel Thank you - your suggested changes have been merged.

**Comment by warmanaMOJ** [comment]
@cybersquirrel Draft 'user guide' ready for final review.

File: guides/user-guide.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+[cyberconsultancy@digital.justice.gov.uk](mailto:cyberconsultancy@digital.justice.gov.uk)
+
+## About the team
+
+The Cyber Security Consulting Team is part of Ministry of Justice Security & Privacy. The consultants are led by the MoJ Chief Information Security Officer, Jon Lawrence.
+
+The team provides help and guidance around cyber security matters, such as:
+
+- Understanding the risks facing your systems and services.
+- Developing services using best practices.
```
Comment (cybersquirrel) [inline guides/user-guide.md]: This needs the word security in it, I think.

File: guides/user-guide.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+To ask for help from a cyber security consultant, send an email to:<br/>
+[cyberconsultancy@digital.justice.gov.uk](mailto:cyberconsultancy@digital.justice.gov.uk)
+
+## About the team
+
+The Cyber Security Consulting Team is part of Ministry of Justice Security & Privacy. The consultants are led by the MoJ Chief Information Security Officer, Jon Lawrence.
+
+The team provides help and guidance around cyber security matters, such as:
+
+- Understanding the risks facing your systems and services.
```
Comment (cybersquirrel) [inline guides/user-guide.md]: ...and designing and implementing effective mitigations to these risks.

File: guides/user-guide.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+If the request is appropriate for the consultants, some initial estimates are immediately carried out, such as:
+
+- Does the request affect many people or services across the MoJ? The wider the range, the greater the impact of the request.
+- Does the request require a small or large number of people, or time, to address? Some requests can be answered promptly by an email. Other requests might require a team of 2 or 3 people to be assigned for a 2 month project.
+- Does the request need certain skills or experience? This helps determine which consultant might be best placed to help.
+
+The answers to these questions help the team to decide whether a consultant can help immediately or not.
+
+If no consultant is available, the request is put on a 'next day refresh', meaning that it is considered again on the next working day. If a request stays in the 'next day refresh' state for too long - typically a week - it is automatically escalated to higher management for a review and decision.
```
Comment (cybersquirrel) [inline guides/user-guide.md]: This para feels very 'internal' - could we not say that our our SLA is to reply to the requestor within one working day of receiving the request, although large / complex requests may require more information and discussion to work out the right way to support?

File: guides/user-guide.md (hunk 1 above)
Comment (cybersquirrel) [reply guides/user-guide.md]: Agree with Joel!

File: guides/user-guide.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+---
+expires: 2019-12-31
+---
+
+# Cyber Security Consulting Team: asking for help
```
Comment (cybersquirrel) [inline guides/user-guide.md]: Style: you appear to have changed the team name from 'consultancy' team to the 'consulting' team.

Comment (warmanaMOJ) [reply guides/user-guide.md]: Updated accordingly.

File: guides/user-guide.md (hunk 4 above)
Comment (warmanaMOJ) [reply guides/user-guide.md]: Updated accordingly.

File: guides/user-guide.md (hunk 3 above)
Comment (warmanaMOJ) [reply guides/user-guide.md]: Added.

File: guides/user-guide.md (hunk 2 above)
Comment (warmanaMOJ) [reply guides/user-guide.md]: Added.

**Comment by warmanaMOJ** [comment]
@cybersquirrel Latest version including comments and feedback to date.

**Pull request merged**